src/sentry/tagstore/snuba/backend.py

```diff
@@ -380,6 +383,128 @@ def get_groups_user_counts(self, project_id, group_ids, environment_id):
                              referrer='tagstore.get_groups_user_counts')
         return defaultdict(int, {k: v for k, v in result.items() if v})
 
+    def get_tag_value_paginator(self, project_id, environment_id, key, query=None,
+            order_by='-last_seen'):
+        from sentry.api.paginator import SequencePaginator
+        from sentry.tagstore.types import TagValue
+
+        if not order_by == '-last_seen':
+            raise ValueError("Unsupported order_by: %s" % order_by)
```

Comment:
The only order_by used in the API is `-last_seen`. I prefer to have the argument there anyway so people can see what's going on but throw an exception so people know if they try something different in the future they'll need to add support.

Reply:
👍